### PR TITLE
Add missing OSPP FIPS remediations

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/ansible/rhel7.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# platform = Red Hat Enterprise Linux 7
+# reboot = false
+# strategy = enable
+# complexity = low
+# disruption = low
+- name: Ensure dracut-fips is installed
```
Comment (jan-cerny) [inline linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/ansible/rhel7.yml]: I think we should use templates instead.

**Comment by dahaic** [comment]
@redhatrises by that you mean you want this to be available only on platforms that are certifiable? I still do not get it. I agree we need specific OVAL, as it has `installed_OS_is_certified`. Remediation though is exactly the same as any other rule.

By not templating those, you just increase the probability of failure.

**Comment by mpreisler** [comment]
I agree with @dahaic and @jan-cerny , we should use custom OVAL  because of installed_OS_is_certified but the fixes can be templated. That way we lower the probability of bugs and get better coverage across fix languages.

**Comment by redhatrises** [comment]
> @redhatrises by that you mean you want this to be available only on platforms that are certifiable? I still do not get it. I agree we need specific OVAL, as it has `installed_OS_is_certified`. Remediation though is exactly the same as any other rule.
> 
> By not templating those, you just increase the probability of failure.

Actually they are no longer the same. As the OVAL checks for a certified OS, so now do the remediations. FIPS content, due to certification, needs to be not be blindly implemented like it has been through templates.

**Comment by jan-cerny** [comment]
Does it try to solve the problem that CentOS and similar derivatives are not FIPS certified? Why should a remediation check for CPE?

File: linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/ansible/rhel7.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# platform = Red Hat Enterprise Linux 7
```
Comment (mpreisler) [inline linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/ansible/rhel7.yml]: use multi_platform_rhel instead please

File: linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/bash/rhel7.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# platform = Red Hat Enterprise Linux 7
```
Comment (mpreisler) [inline linux_os/guide/system/software/integrity/fips/package_dracut-fips_installed/bash/rhel7.sh]: multi_platform_rhel

**Comment by mpreisler** [comment]
Ah, OK. Could you please unify the fix flags (severity, disruption, etc...) and use multi_platform_rhel for all of them? Then I think this can be merged.

**Comment by redhatrises** [comment]
> Ah, OK. Could you please unify the fix flags (severity, disruption, etc...) and use multi_platform_rhel for all of them? Then I think this can be merged.

done

**Comment by shawndwells** [comment]
Requested changes were made. Merging.

**Pull request merged**